scripts/generate_report.sh

```
@@ -3,226 +3,394 @@
-
-    # 数値であるかどうかを確認
-    if ! [[ "$result" =~ ^[0-9]+$ ]]; then
-        log "Error: Result is not a valid number: $result"
-        exit 1
-    fi
-
-    echo "$result"
+    '
+  fi
+
+  local JSON=$(gh api graphql \
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)